numpy/_core/src/common/ufunc_override.c

```diff
@@ -108,7 +108,7 @@ PyUFuncOverride_GetOutObjects(PyObject *kwds, PyObject **out_kwd_obj, PyObject *
          * PySequence_Fast* functions. This is required for PyPy
          */
         PyObject *seq;
-        seq = PySequence_Fast(*out_kwd_obj,
+        seq = PySequence_Fast(*out_kwd_obj, // noqa: borrowed-ref OK
```

Comment:
I actually have some changes I need to upstream related to this from back in February, see https://github.com/numpy/numpy/issues/28046#issuecomment-2683109891. Maybe I should finally get around to sending in what I have...